models/m_payment_mollie_api.erl

```diff
@@ -58,12 +65,13 @@ create(PaymentId, Context) ->
             Recurring =
                 case proplists:get_value(is_recurring_start, Payment) of
                     true ->
-                        [{recurringType, <<"first">>}];
+                        [{sequenceType, <<"first">>}];
                     false ->
                         []
                 end,
```

Comment:
Kan `is_recurring_start` soms ook niet gedefinieerd zijn binnen de Payment, en dat er dan `undefined` terugkomt?

Reply:
`is_recurring_start` is een kolom in de database met als default `false`.  Die komt dus altijd terug uit de `m_payment:get/2`